tasks/audit-cat2.yml

```diff
@@ -92,3 +92,16 @@
       - audit
       - RHEL-07-041010
       - always
+
+- name: "MEDIUM | RHEL-07-021100 | AUDIT | Cron logging must be implemented."
+  command: grep 'cron.* /var/log/cron.log' /etc/rsyslog.conf
```

Comment:
Use the find module to pull these values. 

you can search for patterns inside of directories also containing certain strings and register that output.